vtex-apps/store
# Change events triggered by the store

#### What is the purpose of this pull request?

To change the events the store triggers. 

Some were very specific to the requests of Request Capture, which I considered wrong. The events should be generic and made specific in the app layer, in that case in `request-capture-app`.

#### What problem is this solving?

Fulfil the events of `request-capture-app`.

#### How should this be manually tested?

1. Open https://breno--storecomponents.myvtex.com/
2. Click on a product
3. Click on the "Clothing" menu
4. Click on the "Hats" menu item inside of "Clothing"
5. Type `top` in the search bar
6. Check the events at `window.pixelManagerEvents`

#### Types of changes
